### PR TITLE
feature/ added entity type to submission 

### PR DESCRIPTION
**Pull request by toan-quach**

### CODE REVIEW

**Review by jrobinAV — DISMISSED** [review]
It looks good to me.
We can merge it if we want, but the most important is to do it again in taipy repository once core repo is merged into taipy repo.

@toan-quach Please let us know when the PR is opened on taipy.

Thx!

File: tests/core/job/test_job_manager.py [hunk 1]
```diff
@@ -349,8 +349,8 @@ def test_cancel_subsequent_jobs():
     task_3 = Task("task_config_3", {}, print, [dn_4], id="task_3")
 
     # Can't get tasks under 1 scenario due to partial not serializable
-    submission_1 = submission_manager._create("scenario_id")
-    submission_2 = submission_manager._create("scenario_id")
+    submission_1 = submission_manager._create("scenario_id", "SCENARIO")
```
Comment (gmarabout) [inline tests/core/job/test_job_manager.py]: 🥜 It's really a neat, but maybe we could use `Scenario._ID_PREFIX` instead...

File: src/taipy/core/submission/submission.py [hunk 2]
```diff
@@ -44,13 +44,15 @@ class Submission(_Entity, _Labeled):
     def __init__(
         self,
         entity_id: str,
+        entity_type: str,
```
Comment (gmarabout) [inline src/taipy/core/submission/submission.py:47]: 💬 It means that we can pass any str (i.e. not an enum). 
What should we expect if we pass a string that does not correspond to a real entity?

**Review by gmarabout — DISMISSED** [review]
I left a couple of comments (they are more questions actually), but looks great to me!

File: src/taipy/core/submission/submission.py (hunk 2 above)
Comment (toan-quach) [reply src/taipy/core/submission/submission.py:47]: hmmm good point, as this is only an entity for internal usage, I hope this won't be an issue 😅

File: tests/core/job/test_job_manager.py (hunk 1 above)
Comment (toan-quach) [reply tests/core/job/test_job_manager.py]: ah haha sure thing!

**Review by gmarabout — APPROVED** [review]

**Comment by jrobinAV** [comment]
The code is already merged in taipy repo, right?
Can we close this PR?

**Comment by toan-quach** [comment]
Yep can! Let me close it!

**Pull request closed without merging**